shared/interactive-whiteboard/get-started-uikit/android.mdx

```diff
@@ -188,55 +188,63 @@ To join the Interactive Whiteboard room, do the following:
-            // Gets the fastboard instance
-            val fastboardView = findViewById<FastboardView>(R.id.fastboard_view)
-            val fastboard = fastboardView.fastboard
-
-            // Sets room configuration parameters
-            val roomOptions = FastRoomOptions(
-                "The App Identifier of your whiteboard project",
-                "The room UUID",
-                "The room Token",
-                "The unique identifier of a user",
-                "us-sv"
-            )
-
-            // Creates the FastRoom instance
-            val fastRoom = fastboard.createFastRoom(roomOptions)
-            // Joins the whiteboard room
-            fastRoom.join()
-        }
+``` java
+package com.example.fastboardquickstart
+
+import android.content.pm.ActivityInfo
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import android.widget.Button
+import io.agora.board.fast.FastboardView
+import io.agora.board.fast.model.FastRegion
+import io.agora.board.fast.model.FastRoomOptions
+import java.lang.Exception
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        // Sets the screen orientation as landscape
+        supportActionBar?.hide()
+        requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+         setupFastboard()
+
     }
-    ```
+
+        private fun setupFastboard() {
+        // Gets the fastboard instance
+        val fastboardView = findViewById<FastboardView>(R.id.fastboard_view)
+        val fastboard = fastboardView.fastboard
+
+        // Sets room configuration parameters
+        val roomOptions = FastRoomOptions(
+            "The App Identifier of your whiteboard project",
+            "The room UUID",
+            "The room Token",
+            "The unique identifier of a user",
+           FastRegion.US_SV
+        )
+
+        // Creates the FastRoom instance
+        val fastRoom = fastboard.createFastRoom(roomOptions)
+        // Joins the whiteboard room
+        fastRoom.join()
+    }
+}
+```
 
 ## Build and run the project
 
 Build the project in Android Studio, and run it on a simulator or a physical mobile device. If the project runs successfully, you can see the following page:
 
 ![](https://web-cdn.agora.io/docs-files/1656648527191)
 
-You can select the pencil, text, or shape tools on the left toolbar to write and draw on the whiteboard. When the current page is full, you can click the ![](https://web-cdn.agora.io/docs-files/1656648545342) icon at the bottom to add a new page, and use the arrows to switch pages.
+You can select the pencil, text, or shape tools on the left toolbar to write and draw on the whiteboard. When the current page is full, click the `+` icon in the center of the screen to add a new page; use the arrows to switch pages.
```

Comment:
There are icons in quickstarts of other platforms, other guides, and also, other products that needs to be resolved. 
Rewriting the icons into symbols or texts sometimes might not be easy, such as the icon of adding plugins in the "Whiteboard Web get-started-uikit". Also, locating and rewriting all the icons could cost great human resources. I believe a one-size-fits-all solutions of rendering icons is needed.